otcextensions/sdk/elb/v2/lb_certificate.py

```diff
@@ -0,0 +1,56 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+from openstack import resource
+
+
+class Certificate(resource.Resource):
+    resource_key = 'certificate'
+    resources_key = 'certificates'
+    base_path = ('/lbaas/certificates')
+
+    # capabilities
+    allow_create = True
+    allow_fetch = True
+    allow_commit = True
+    allow_delete = True
+    allow_list = True
+
+    _query_mapping = resource.QueryParameters(
+        'marker', 'limit',
```

Comment:
Please add the other query params of: https://docs.otc.t-systems.com/api/elb/elb_zq_zs_0002.html to ensure that name, id etc. is also evailable.